ckanext/stcndm/logic/common.py

```diff
@@ -94,6 +106,29 @@ def get_autocomplete(context, data_dict):
 
 # noinspection PyIncorrectDocstring
 @logic.side_effect_free
+def get_internal_authors(context, data_dict):
+    q = _get_or_bust(data_dict, 'q')
+    engine = ds_db._get_engine({
+        'connection_url': config['ckan.datastore.write_url']
+    })
+    session = orm.scoped_session(orm.sessionmaker(bind=engine))
+    results = (
+        session.query(
+            internal_authors.c.first_name,
+            internal_authors.c.last_name
+        )
+        .filter(
+            internal_authors.c.full_name.ilike('%{q}%'.format(q=q))
+        )
+        .limit(25)
+        .all()
+    )
+    session.remove()
+    return results
```

Comment:
@TkTech what is the point of redefining the table and doing all this work instead of using a couple-line call to datastore_search_sql?